build-ps/ubuntu/percona-server-common-5.7.postinst

```diff
@@ -7,13 +7,23 @@ case "$1" in
     # New packaging paradigm for my.cnf as of Dec-2014 for sharing mysql
     # variants in Ubuntu. If the new mysql-common package does not provide
     # the update-alternatives facility, fall back to creating a symlink
-    if [ -f /usr/share/mysql-common/configure-symlinks ]
+#    if [ -f /usr/share/mysql-common/configure-symlinks ]
+#    then
+#      /usr/share/mysql-common/configure-symlinks install percona-server "/etc/mysql/percona-server.cnf"
+#    else
+#      echo "Notice: configure-symlinks trigger could not be called."
+#      echo "Please manually make /etc/mysql/my.cnf a symlink that points to percona-server.cnf."
+#    fi
+    if [ ! -f /etc/mysql/my.cnf ]
     then
-      /usr/share/mysql-common/configure-symlinks install percona-server "/etc/mysql/percona-server.cnf"
+      update-alternatives --force --install /etc/mysql/my.cnf my.cnf "/etc/mysql/percona-server.cnf" 200
     else
-      echo "Notice: configure-symlinks trigger could not be called."
-      echo "Please manually make /etc/mysql/my.cnf a symlink that points to percona-server.cnf."
-    fi  
+      echo " -------------"
+      echo "   *  The suggested my.cnf options and settings are in percona-server.conf.d/mysqld.cnf"
```

Comment:
should this be: "*  The suggested my.cnf options and settings are in /etc/percona-server.conf.d/my.cnf"?